AWSAuthSDK/AWSAuthSDKTestAppUITests/AWSHostedUIUserPoolTests.swift

```diff
@@ -58,34 +55,70 @@ class AWSHostedUIUserPoolTests: XCTestCase {
         // Check if all user details are present
         inspectTokenDetails(application: app)
         inspectCredentialDetails(application: app)
+    }
+
+    /// Test successful refresh access token and get user attributes
+    ///
+    /// - Given: Invalidate the access token of an user after signing in
+    /// - When:
+    ///    - I try to get user attributes after access token is invalidated
+    /// - Then:
+    ///    - I should successfully get user attributes
+    ///
+    func testHostedUIGetAttributesWhenAccessTokenInvalidated() {
+        let username = "accessToken" + UUID().uuidString
+        let app = XCUIApplication()
+        signOutUserpool(application: app)
         
-        // Initiate attribute update
-        app.buttons["Update User Attribute"].tap()
+        signUpAndVerifyUser(username: username)
         
-        // Check again if the values are present
+        // Push the hosted UI view controller
+        app.buttons["Hosted UI Userpool tests"].tap()
+        XCTAssertTrue(app.navigationBars["UserPool"].exists)
+        
+        //Initiate signIn
+        app.buttons["SignIn User"].tap()
+        signInUserpool(application: app, username: username)
+        
+        // Check if successfully signed in
+        let userPoolSignInStateLabelElement = app.staticTexts["userPoolSignInStateLabel"]
+        let predicate = NSPredicate(format: "label CONTAINS[c] %@", "signedIn")
+        let expectation1 = expectation(for: predicate,
```

Comment:
expectation1 remove 1

Reply:
The complier returns error `Variable used within its own initial value`
See line 86, the name of function is called `expectation()`, so the variable cannot be called expectation